### PR TITLE
Fix to developer certificates not being entirely returned

### PR DESCRIPTION
**Pull request by thegecko**
@arekzaluski @madchutney this resolves IOTSTOOL-688

I also took the liberty of including a small enhancement for an automatic web page launch when starting the proxy test server. I can remove that if it confuses anything.

### CODE REVIEW

File: src/certificates/certificatesApi.ts [hunk 1]
```diff
@@ -64,12 +64,24 @@ export class CertificatesApi {
     }
 
     private extendCertificate(iamCert: iamCertificate, done: (error: any, certificate: any) => any) {
+        if (iamCert.device_execution_mode === 1) {
```
Comment (arekzaluski) [inline src/certificates/certificatesApi.ts]: 👍  My code to fix it look exactly the same. I only have if else

File: examples/proxy/server.js [hunk 2]
```diff
@@ -130,4 +131,13 @@ app.use(apiPath, (req, res, next) => {
...
 // Start server
 http.createServer(app)
-.listen(port, () => console.log(`Proxy server listening on port ${port}`));
+.listen(port, () => {
+    var url = `http://localhost:${port}`;
+    console.log(`Proxy server listening at ${url}`);
+
+    var cmd = path.join(__dirname, "xdg-open");
+    if (process.platform === "darwin") cmd = "open";
+    else if (process.platform === "win32") cmd = `start ""`;
```
Comment (bogthe) [inline examples/proxy/server.js]: Should it not just be `start` instead of `start ""` ?

File: src/certificates/endpoints.ts [hunk 3]
```diff
@@ -21,23 +21,18 @@ import {
     AccountAdminApi as AdminApi,
     DeveloperApi as AccountDeveloperApi
 } from "../_api/iam";
-import {
-    ServerCredentialsApi as ServerApi,
-    DeveloperCertificateApi as CertDeveloperApi
-} from "../_api/connector_ca";
+import { ExternalAPIApi as ConnectorApi } from "../_api/connector_ca";
```
Comment (arekzaluski) [inline src/certificates/endpoints.ts]: That surprise me. In Python SDK `ServerCredentialsApi` still exists. It's strange that now it is generated in JS as `ExternalAPIApi`

File: examples/proxy/server.js (hunk 2 above)
Comment (thegecko) [reply examples/proxy/server.js]: No, windows takes the title as the first parameter.

File: src/certificates/endpoints.ts (hunk 3 above)
Comment (thegecko) [reply src/certificates/endpoints.ts]: There are 3 APIs in the JS SDK, ExternalAPIApi replicates all the others, so I switched to use just that one :)

File: src/certificates/certificatesApi.ts (hunk 1 above)
Comment (thegecko) [reply src/certificates/certificatesApi.ts]: :)

**Pull request merged**